web/client/examples/viewer/index.html

```diff
@@ -25,6 +25,16 @@
             padding: 0;
             overflow: hidden;
         }
+        #snapshot_map{
```

Comment:
I think this should stay in the css of the component that initialize it, included with require. 
we should also use something like "snapshot_hidden_map"